apis/v1alpha1/httproute_types.go

```diff
@@ -247,9 +268,8 @@ type HTTPRouteMatch struct {
 	// Path specifies a HTTP request path matcher. If this field is not
 	// specified, a default prefix match on the "/" path is provided.
 	//
-	// +optional
 	// +kubebuilder:default={type: "Prefix", value: "/"}
-	Path *HTTPPathMatch `json:"path"`
+	Path *HTTPPathMatch `json:"path,omitempty"`
```

Comment:
`Path` should be a value instead of a pointer when the field is being defaulted.

Reply:
Updated, thanks!